src/main/java/coverage2/Home.java

```diff
@@ -3,6 +3,7 @@
 public class Home {
     private String name;
     private String address1;
+    private String address2;
     
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube2.tools.kmb.sonynei.net/coding_rules#rule_key=squid%3AUndocumentedApi)